### ISSUE TITLE
Milestone - Enhanced support for metabolomics or metaproteomics standards and controlled vocabulary (2.4)

### ISSUE THREAD

**Issue by ssarrafan** [microbiomedata/issues#448]
We aim to engage with the broader metabolomics and metaproteomics research communities, and
relevant standards organizations by hosting NMDC community workshops to encourage discussions
around identifying gaps, challenges, and next steps necessary to update and modernize many of the
existing standards in this space (Milestone 2.3). To represent the study design and chemistry techniques,
we will adopt metadata elements from existing standards and expand the NMDC schema (Milestone 2.4).
These standards include mzML65 from the PSI Mass Spectrometry Standards Working Group and the
Metabolomics Standards Initiative Ontology (MSIO)66. We will also work closely with organizations such
as IMMSA67 and MANA68, in an analogous fashion to how we worked with the GSC (see Letter of Support).
For the quantification results, we will reuse metadata elements from the mzTab69 standard, and aim to
reconcile all metadata element definitions with International Union of Pure and Applied Chemistry
(IUPAC)70 definitions and the applicable elements of the MIxS standard. This will ensure common physical
sample reconciliation, reproducible results, and accurate interpretation of analytical workflow outputs.

Pages 28-29

**Comment by mslarae13** [comment]
@anastasiyaprymolenna  you should add this milestone to your metabolink issues

**Comment by ssarrafan** [comment]
This is now a Q4 milestone.

**Comment by ssarrafan** [comment]
@lamccue this milestone is due this quarter.  Will this be done by September?  Thank you

**Comment by mslarae13** [comment]
> We aim to engage with the broader metabolomics and metaproteomics research communities, and
relevant standards organizations by hosting NMDC community workshops to encourage discussions
around identifying gaps, challenges, and next steps necessary to update and modernize many of the
existing standards in this space (Milestone 2.3). 

Kinda, we did some MANA and IMMSA workshops in 2023? I think it was.

> To represent the study design and chemistry techniques, we will adopt metadata elements from existing standards and expand the NMDC schema (Milestone 2.4). These standards include mzML65 from the PSI Mass Spectrometry Standards Working Group and the Metabolomics Standards Initiative Ontology (MSIO)66.

Yup, this is berkeley-schema. But, IDK about mzML? We did use terms from MSIO.

> We will also work closely with organizations such as IMMSA67 and MANA68, in an analogous fashion to how we worked with the GSC (see Letter of Support). For the quantification results, we will reuse metadata elements from the mzTab69 standard, and aim to reconcile all metadata element definitions with International Union of Pure and Applied Chemistry
(IUPAC)70 definitions and the applicable elements of the MIxS standard. This will ensure common physical
sample reconciliation, reproducible results, and accurate interpretation of analytical workflow outputs.

Nope! Least, not yet. Not really ready. And not that I'm aware of.

**Comment by lamccue** [comment]
@mslarae13 
We marked Milestone 2.3 done earlier this year. It was a vague milestone to have a _strategy_ for working with the mass spec working groups. In addition to the activities with IMMSA and MANA, we connected with the metaproteomics initiative. You'll meet Tim next week :)

@ssarrafan  
Yes, we can mark Milestone 2.4 done this quarter. As Montana indicated, the metadata elements are in berkeley-schema. Note that the describing text from the proposal hasn't aged well. We did end up adopting terms that we could from other sources, but we with regard to mzML, it is a data format and doesn't establish metadata standards. So we can't hold to that old (poorly crafted) text literally.

As to that last part of the old proposal text, it is very aspirational. We **aim** "to reconcile all metadata definitions with other sources. That is a great aspiration. Would be an impossible milestone, and I'm glad we didn't try to assign a milestone to it. Worth considering what we should propose to do with regard to mapping to IUPAC or other systems' metadata definitions in the renewal, now that we are more knowledgeable about the current state of some of the other efforts.
